### PR TITLE
ANC Logic - merge Recommended Treatment into Medication Distribution

### PR DESCRIPTION
**Pull request by anvmn**
#255 
#206 
#222

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-04-26 14-58

**Comment by anvmn** [comment]
Selection_685

**Comment by anvmn** [comment]
Selection_686

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -5635,6 +5658,31 @@ translationSet trans =
...
+            , kinyarwanda = Nothing
+            }
+
+        MedicationDistributionHelperDiscordantPartnershipNoARVs ->
+            { english = "This patient is part of a discordant partnership in which the partner is not on ARVs"
+            , kinyarwanda = Nothing
+            }
+
+        MedicationDistributionHelperHIV ->
+            { english = "This patient is HIV postive"
```
Comment (mariano-dagostino) [inline client/src/elm/Translate.elm]: @anvmn typo here.

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good a minor typo in a string.

**Pull request merged**